src/App.jsx

```diff
@@ -1,3 +1,33 @@
+import { useEffect, useState } from "react";
+import { fetchTeams } from "./services/teams";
+import style from './App.css';
+import Home from "./views/Home/Home";
+import { Switch, Route } from "react-router-dom";
+import TeamDetail from "./views/Team/TeamDetail";
+
 export default function App() {
-  return <h1>Hello World</h1>;
+  const [teams, setTeams] = useState([]);
+  
+  useEffect(() => {
+    const getData = async () => {
+      const data = await fetchTeams();
+      setTeams(data);
+    }
+    getData();
+  }, []);
+
```

Comment:
typically you will want to keep state out of App.jsx and place it in your Views.